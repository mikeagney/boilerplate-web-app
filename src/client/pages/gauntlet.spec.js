import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Gauntlet from './gauntlet';

describe('Gauntlet page component', () => {
  it('will render the Gauntlet page', () => {
    // Arrange
    const wrapper = shallow(<Gauntlet />);

    // Act
    // Assert
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
