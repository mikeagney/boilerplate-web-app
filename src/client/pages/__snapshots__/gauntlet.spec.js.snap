// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gauntlet page component will render the Gauntlet page 1`] = `
<div>
  <h2>
    Gauntlet
  </h2>
</div>
`;
